src/Control/Effect/Labelled.hs

```diff
@@ -70,6 +77,35 @@ instance Algebra (eff :+: sig) (sub m) => Algebra (Labelled label eff :+: sig) (
     R sig -> Labelled . alg (runLabelled . hdl) (R sig)
   {-# INLINE alg #-}
 
+type HasLabelledLift n sig m = HasLabelled Lift (Lift n) sig m
```

Comment:
Can you eta-reduce this definition? (It has ramifications w/r/t partial application of type synonyms).

Reply:
Redefine as: type HasLabelledLift n sig m = (LabelledMember Lift (Lift n) sig, Algebra sig m)